tasks/copy.js

```diff
@@ -0,0 +1,21 @@
+var gulp = require('gulp');
+var copy = require('gulp-contrib-copy');
+var srcDir = 'src';
+var appDir = srcDir + '/app';
+
+gulp.task('copy', function() {
```

Comment:
We need to separate the concept of macro tasks and micro tasks. We need to discuss about it.